File: Object-oriented-programming/Lab 1.2/C#/Program.cs
using System;
using Dale;

namespace Chip {

public class Program {
    public static void Main()
    {
        Program program = new Program();
        program.Input();
    }

        public void Input()
        {
            Console.Write("Type string: ");
            string input = Console.ReadLine() ?? "";

            MainClass transfer = new MainClass(input);
            MainClass transfer2 = new MainClass(transfer);
            transfer.Main();
    }

    public void Output(string input, int input_Length, string input_Lowercase, string input_Sorted)
    {
        Console.WriteLine("");
        Console.WriteLine("Input: " + input);
        Console.WriteLine("Lowercase: " + input_Lowercase);
        Console.WriteLine("Length: " + input_Length);
        Console.WriteLine("Sorted: " + input_Sorted);
    }
  }
}

File: Object-oriented-programming/Lab 1.2/C++/Program.cpp
#include <iostream>
#include <string>
#include "String.h"

int main() {
    std::string input;

    std::cout << "Type string: ";
    std::getline(std::cin, input);

    StringManipulator transfer;

    std::cout << "Input: " << input << std::endl;
    std::cout << "Lowercase: " << transfer.toLowercase(input) << std::endl;
    std::cout << "Length: " << transfer.getLength(input) << std::endl;
    std::cout << "Sorted: " << transfer.sortString(input) << std::endl;
    return 0;
}
